Directory.Packages.props

```diff
@@ -10,14 +10,14 @@
 
   <PropertyGroup Condition=" '$(TargetFramework)' == 'net8.0'">
     <FrameworkVersion>8.0.1</FrameworkVersion>
-    <ExtensionsVersion>8.0.0</ExtensionsVersion>
+    <ExtensionsVersion>9.0.3</ExtensionsVersion>
```

Comment:
Do we need to change this? I would be in favor of cleaning up the dependencies separately from other changes: I think maybe ExtensionsVersion shouldn't exist at all, since the Microsoft.Extensions packages don't all release in sync.

Reply:
Good discussion point. The HybridCache has quite a number of dependencies at a minimum to 9.0.3. So in this case, it does make sense..